src/components/MetadataEditor/MetadataEditor.vue

```diff
@@ -0,0 +1,383 @@
+<template>
```

Comment:
Sorry about the changes & move of this file occurring in the same PR. GitHub isn't very good at tracking files that have moved, so it makes it look like the old version was deleted and this was added. Here's a better diff: https://www.diffchecker.com/xsnITcul